### ISSUE TITLE
Implementation of complete message

### ISSUE THREAD

**Issue by LostLogia4** [LostLogia4/KancolleLoader#18]
When the loading is completed, the textbox would be changed to "complete".

**Comment by LostLogia4** [comment]
Reference code

```
$(window).bind("load", function() {
    $("#progress-text").text("Loading complete");
});
```

**Comment by sinsinpub** [comment]
is that the html page loading? if not, may not bind `onload` on `window`...

**Comment by LostLogia4** [comment]
@sinsinpub It does when it's loaded from the get-go, but probably not when it's from the buttons (which appears when the expiry date is past due)

> bind `onload` on `window`

how do I do that?

**Comment by sinsinpub** [comment]
is the `load complete` event supposed to be fired on DOM modification by AJAX after the page init loading?

if true, see https://stackoverflow.com/questions/13346512

**Comment by LostLogia4** [comment]
@sinsinpub no, it's supposed to be fired after DOM modification **and** after all elements is loaded. However, this is not a one-time event, it may happen once or repeated several times, so when one of thoe button is clicked it's not supposed to stay in "load completed" state.

**Comment by LostLogia4** [comment]
I'll keep this one for reference:
Requires asynchronous Jquery function such as 
`$.Deffered() and $.when()`
